stdlib/experimental/usage/usage.flux

```diff
@@ -0,0 +1,48 @@
+package usage
+
+import "experimental/influxdb"
+import "csv"
+import "experimental/json"
+
+// from returns an organization's usage data. The time range to query is
+// bounded by start and stop arguments. Optional orgID, host and token arguments
+// allow cross-org and/or cross-cluster queries. Setting the raw parameter will
+// return raw usage data rather than the downsampled data returned by default.
+//
+// Note that unlike the range function, the stop argument is required here,
+// pending implementation of https://github.com/influxdata/flux/issues/3629.
+from = (start, stop, host="", orgID="{orgID}", token="", raw=false) => {
+	response = influxdb.api(
+        method: "get",
+		path: "/api/v2/orgs/" + orgID + "/usage",
```

Comment:
We probably need a path sanitation method here otherwise the orgID would allow path expressions like `../../`

Reply:
I don't think that's a concern, since any request to the API has to be authorized anyway. If a user abused the org ID parameter in this way, they would not be authorized. Also, using the underlying `influxdb.api` function would avoid any sanitation we attempted, as would `http.get` or `http.post`.

Reply:
What about cases where another bit of Flux code is calling the `usage.from` code? It would allow the caller of `usage.from` to actually call another API endpoint. While the API will still implement its own auth it still is a concern that calling code can break the intent of the function. 

We have the `http.pathEscape` function which can be applied here to make it safe.